addons/account/static/src/components/duplicate_bill_warninng/duplicate_bill_warning.js

```diff
@@ -0,0 +1,69 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+import { Component } from "@odoo/owl";
+import { getCurrency } from "@web/core/currency"
+
+
+class DuplicateBillWarning extends Component {
+    static template = "account.DuplicateBillWarning";
+    static props = {};
+    setup() {
+        this.action = useService("action");
+        this.orm = useService("orm");
+    }
+
+    async discard() {
+        if (this.props.record.isInEdition) {
+            await this.props.record.discard();
+        }
+    }
+
+    async open_tree_and_discard() {
+        await this.discard();
+        const action = await this.orm.call("account.move", "open_duplicated_ref_bill_view", [this.props.record.resId], {});
+        this.action.doAction(action);
+    }
+
+
+    async open_form_and_discard(id) {
+        await this.discard();
+        this.action.doAction({
+            type: "ir.actions.act_window",
+            res_model: this.current_field.resModel,
+            res_id: id,
+            views: [[false, "form"]],
+            view_mode: "form",
+        });
+    }
+
+    get records_count() {
+        return this.current_field.count;
+    }
+
+    get records_data() {
+        return this.current_field.records.map((bill) => {
+            return {
+                name: bill.data.display_name,
+                id: bill.resId,
+                partner_name: bill.data.partner_id[1],
+                total: `${bill.data.amount_total} ${getCurrency(bill.data.currency_id[0]).symbol}`
```

Comment:
Just use `display_name` with `input_full_display_name` in the context of the field
And add the amount in the computation of the field in that case (see `_get_move_display_name`)